client/modules/Rewards/components/RewardTypes.scss

```diff
@@ -0,0 +1,42 @@
+@import '~styles/variables';
+
+.rewardTypes {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-around;
+	width: 100%;
+}
+
+.type {
```

Comment:
Put this within `.rewardTypes` for now, since most of this is component specific.